### PR TITLE
LUN-13: Refactor css for screens fixes

### PR DESCRIPTION
**Pull request by YasminNajar**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/routes/ResultsOverviewScreen.tsx [hunk 1]
```diff
@@ -1,85 +1,85 @@
...
   align-self: center;
-  padding-top: 17;
-  padding-bottom: 17;
-  padding-right: 8;
-  padding-left: 16;
-  margin-bottom: 8;
+  padding-top: 17px;
+  padding-bottom: 17px;
+  padding-right: 8px;
+  padding-left: 16px;
```
Comment (Taggotty) [inline src/routes/ResultsOverviewScreen.tsx:60]:
```suggestion
  padding: 17px 8px 17px 16px
```

File: src/routes/write-exercise/components/AnswerSection.tsx [hunk 2]
```diff
@@ -16,34 +16,34 @@ import AudioPlayer from '../../../components/AudioPlayer'
...
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-width: 1px;
+  border-radius: 2px;
+  padding-right: 15px;
+  padding-left: 15px;
+  margin-top: 12%;
+  margin-bottom: 12%;
+  border-color: ${(prop: StyledProps) => prop.StyledBorder};
```
Comment (Taggotty) [inline src/routes/write-exercise/components/AnswerSection.tsx:38]:
```suggestion
const TextInputContainer = styled.View<{ styledBorderColor: string}>`
  width: 80%;
  height: 16%;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  border-width: 1px;
  border-radius: 2px;
  padding-right: 15px;
  padding-left: 15px;
  margin-top: 12%;
  margin-bottom: 12%;
  border-color: ${prop => prop.styledBorderColor};
```

**Review by Taggotty — CHANGES_REQUESTED** [review]

**Comment by SteffiStuffel** [comment]
Why are the fixes in separate PRs?

**Comment by Taggotty** [comment]
> Why are the fixes in separate PRs?

@YasminNajar told me you told them to do them in a separate PR 😅

**Review by Taggotty — APPROVED** [review]

**Comment by YasminNajar** [comment]
> 
> 
> Why are the fixes in separate PRs?

@SteffiStuffel We were informed by Aya that it's better to do a new branch for it, and Aya I think misunderstood something Steffi said, so I also thought that Steffi told Aya. It might have been a big misunderstanding

**Pull request merged**